resources/testnet-releases.md

```diff
@@ -3,7 +3,7 @@
 
 Variable  | Value
 ------------- | -------------
-`axelar-core` version | `v0.13.7`
+`axelar-core` version | `v0.13.6`
 `tofnd` version | `v0.8.2`
 Ethereum Axelar Gateway contract address | `0x81b450CA8F0e842EcC361FD64270aeb5A374A0d8`
```

Comment:
Need to update contract addresses

Reply:
should probably merge now and do contract addresses in a separate PR so people can join testnet immediately.

Reply:
Adding the ethereum/terra values since we need to set it up